src/components/layouts/signup-layout/style.less

```diff
@@ -0,0 +1,15 @@
+@import '../../../styles/vars.less';
+@import '../../../styles/mixins.less';
+@import '../../../styles/fonts.less';
+
+.SignupLayout {
+  /* Fix bug where ant row overflows container */
+  .ant-row-flex {
+    margin-right: 0 !important;
```

Comment:
Do we need to !important to override ant-row-flex?

Reply:
it’s supposedly a bug at the moment when using gutter and the ant row flex. Didn’t find another solution at the moment